autodoc/admin-api/data/admin-api.lua

```diff
@@ -988,6 +988,8 @@ return {
             match if present in the request.
             The `Host` header cannot be used with this attribute: hosts should be specified
             using the `hosts` attribute.
+            Since version 2.8.0, when `headers` contains only one value and that value starts with
+            the special prefix `~*`, the value will be interpreted as a regular expression.
```

Comment:
```suggestion
            When `headers` contains only one value and that value starts with
            the special prefix `~*`, the value is interpreted as a regular expression.
```
You don't need to add a "starting with 2.8.0", since the docs themselves are versioned.